stripes-web/src/main/java/org/stripesframework/web/controller/UrlBindingFactory.java

```diff
@@ -255,24 +282,6 @@ public int compare( String a, String b ) {
     * @param binding the URL binding
     */
    public void addBinding( Class<? extends ActionBean> beanType, UrlBinding binding ) {
-      /*
-       * Search for a class that has already been added with the same name as the class being
-       * added now. If one is found then remove its information first and then proceed with adding
-       * it. I know this is not technically correct because two classes from two different class
-       * loaders can have the same name, but this feature is valuable for extensions that reload
-       * classes and I consider it highly unlikely to be a problem in practice.
-       */
-      Class<? extends ActionBean> existing = null;
-      for ( Class<? extends ActionBean> c : _classCache.keySet() ) {
-         if ( c.getName().equals(beanType.getName()) ) {
-            existing = c;
-            break;
-         }
-      }
-      if ( existing != null ) {
-         removeBinding(existing);
-      }

```

Comment:
Don't we need to keep this for the `ReloadActions` run configurations?

Reply:
I moved this now to the ActionResolver, this should be enough since its only necessary upon adding or removing whole actions

Reply:
Reloading Actions should now work again - despite Conflict Resolution. This was already broken before and is trickier to fix (see added comment) 
![grafik](https://github.com/Chrono24/stripes/assets/12140954/901bd67b-4c58-47cc-8a6a-2a0f0bcf5e64)